### PR TITLE
PG-1285 Customizable colors to fit the customer demand, logotype etc.

### PR DESCRIPTION
**Pull request by dim2k2006**

### CODE REVIEW

File: apps/queue/customize-booking.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 You can customize the appearance of the booking page by using CSS. 
 
-Enter your custom CSS into the `Add brand guidelines with CSS to your customers booking UI` input field inside the Settings of your Queue. 
+Enter your custom CSS into the `Add brand guidelines with CSS to your queue UI` input field inside the Settings of your Queue.
```
Comment (alex1815) [inline apps/queue/customize-booking.md:5]: queue -> queue solution

File: apps/queue/customize-draft-ticket.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Customize the appearance of the Draft ticket Page
```
Comment (alex1815) [inline apps/queue/customize-draft-ticket.md:1]: of the Confirmed Booking ticket Page

File: apps/_sidebar.md [hunk 3]
```diff
@@ -8,6 +8,7 @@
   * [Setting up a queue](/apps/queue/setting-up)
   * [Order Pickup](/apps/queue/order-pickup)
   * [Customize Booking Page](/apps/queue/customize-booking)
+  * [Customize Draft ticket Page](/apps/queue/customize-draft-ticket)
```
Comment (alex1815) [inline apps/_sidebar.md:11]: Customize Confirmed Booking ticket Page

**Review by alex1815 — APPROVED** [review]

File: apps/queue/customize-booking.md (hunk 1 above)
Comment (dim2k2006) [reply apps/queue/customize-booking.md]: fixed

File: apps/queue/customize-draft-ticket.md (hunk 2 above)
Comment (dim2k2006) [reply apps/queue/customize-draft-ticket.md]: fixed

File: apps/_sidebar.md (hunk 3 above)
Comment (dim2k2006) [reply apps/_sidebar.md]: fixed

File: apps/queue/customize-draft-ticket.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+Draft ticket page is the page that the user sees after booking an appointment.
```
[the remark this answers is not in the document]
Comment (dim2k2006) [reply apps/queue/customize-draft-ticket.md]: fixed

**Pull request merged**